### ISSUE TITLE
Files dates matching Mirth's Last Modified

### ISSUE THREAD

**Issue by carm-afea** [SagaHealthcareIT/mirthsync#35]
It would be great if local files modification datetimes matched the metadata/lastModified from the channel itself.

**Plus**: this would call for an update mode:
- when pulling, allows overwriting for local files that are older than the copy on Mirth, preserving files that where locally modified (thus newer)
- when pushing, allows overwriting remote channels only if our local copy is newer, preserving channels that where remotelly modified, and also preserving their remote lastUpdate in case no changes were made (locally and remotely).

**Comment by jessesaga** [comment]
Yes, I agree. This is similar to the enhancement described in #26. I think both should be implemented at the same time.
